features/support/rails.rb

```diff
@@ -8,6 +8,16 @@
 require "cucumber/rails/database/deletion_strategy"
 require "cucumber/rails/database/shared_connection_strategy"
 require "cucumber/rails/database/truncation_strategy"
+
+# FIXME: NullStrategy is a new strategy introduced in a cucumber-rails version
+# that has not been released yet. Remove this begin-rescue block and bump
+# the cucumber-rails dependency in other gemfiles when a stable version will
+# be released
+begin
+  require "cucumber/rails/database/null_strategy"
+rescue LoadError
+end
+
```

Comment:
@deivid-rodriguez since `cucumber-rails` 2.5 comes with a new strategy that should be required, I'm asking if I should make another PR to point `cucumber-rails` to master in all the gemfiles, so I can remove the conditional check

Reply:
Yes, sure!

Reply:
done in #7366